xml/ay_bigfile.xml

```diff
@@ -17169,6 +17167,9 @@ echo -n 100
          <literal>autoyast=https://..</literal>,
          <literal>autoyast=tftp://..</literal>, or
          <literal>autoyast=cifs://..</literal>.
+         In order to have network access the network will have to be defined via the
+         boot options in linuxrc. One simplest way would to do define it via DHCP like:
```

Comment:
A simple way would be to define...

(Or "**The** simplest..." if you like. But there can be only **one** simplest way by definition.)